src/Components/Common/FilePreviewDialog.tsx

```diff
@@ -30,6 +37,8 @@ type FilePreviewProps = {
   className?: string;
   titleAction?: ReactNode;
   fixedWidth?: boolean;
+  uploadedFiles?: Array<FileUploadModel>;
```

Comment:
Is there a reason for this being nullable?

```suggestion
  uploadedFiles: FileUploadModel[];
```

Reply:
[```src/Utils/useFileManager.tsx```](https://github.com/coronasafe/care_fe/blob/develop/src/Utils/useFileManager.tsx) also uses ([```FilePreviewDialog.tsx```](https://github.com/coronasafe/care_fe/pull/8361/files/faf8e07ed820c584a8b8b5f6e473e7609b59250e#diff-54c239f5693dcce454bb02b533a03b41fc9d4c5998bf3b1ce0aef9fafbe5ceb2)) in there we don't pass uploadedFiles